controls/docker_host_os_level1.rb

```diff
@@ -68,28 +70,39 @@
   impact 1.0
   title 'Harden the container host'
   desc 'Containers run on a Linux host. A container host can run one or more containers. It is of utmost importance to harden the host to mitigate host security misconfiguration'
+  tag 'host'
   ref 'http://dev-sec.io'
+  ref 'https://docs.docker.com/articles/security/'
   ref 'https://docs.docker.com/engine/security/security/'
+  ref 'https://benchmarks.cisecurity.org/downloads/multiform/index.cfm'
+  ref 'http://docs.docker.com/articles/security/#other-kernel-security-features'
+  ref 'https://grsecurity.net/'
+  ref 'https://en.wikibooks.org/wiki/Grsecurity'
+  ref 'https://pax.grsecurity.net/'
+  ref 'http://en.wikipedia.org/wiki/PaX'
 end
 
 control 'cis-docker-benchmark-1.4' do
   impact 1.0
   title 'Remove all non-essential services from the host'
   desc 'Ensure that the host running the docker daemon is running only the essential services.'
+  tag 'host'
   ref 'https://blog.docker.com/2013/08/containers-docker-how-secure-are-they/'
 end
 
 control 'cis-docker-benchmark-1.5' do
   impact 1.0
   title 'Keep Docker up to date'
   desc 'The docker container solution is evolving to maturity and stability at a rapid pace. Like any other software, the vendor releases regular updates for Docker software that address security vulnerabilities, product bugs and bring in new functionality.'
+  tag 'host'
+  ref 'https://docs.docker.com/installation/'
   ref 'https://github.com/docker/docker/releases/latest'
 
   docker_server_version = command('docker version --format \'{{.Server.Version}}\'').stdout
```

Comment:
This would result in
```
describe docker.version do
  it { should cmp >= '1.13.1'}
end
```
then. We can use this syntax, once the PR has been merged. We should not block this PR